package.json

```diff
@@ -625,6 +649,31 @@
               "trace"
             ],
             "default": "info"
+          },
+          "dfdlDebuggerLogLevel": {
```

Comment:
Concern: I think these schema properties need to match the new hierarchical structure.

Reply:
Or at least it is confusing to have them not match, I see these names used [here](https://github.com/apache/daffodil-vscode/pull/694/files#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351R128).

Reply:
So those ones are just used for default values of the hierarchy items. It is the same thing we do with other hierarchy items. It might be something to make a new ticket for this, instead of making these items like we do currently for each hierarchy item individually we just make one object default item that sets all of them at once. Does sound good? Since, this is done in multiple places, it would be good to have a separate PR to fix this everywhere.

Reply:
@arosien Created issue https://github.com/apache/daffodil-vscode/issues/698 to fix these items in a different PR.